Assets/Scripts/Buddy/BehaviorPhysicsController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[RequireComponent( typeof( ActorPhysics ) )]
+public class BehaviorPhysicsController : MonoBehaviour
+{
```

Comment:
Do we need/want a falling state in here for animation control? Level design could just make this not happen, I guess but that limits the garden somewhat.